workflows/templates/base/ssh.template.argo.yaml

```
@@ -65,6 +65,7 @@ spec:
           else
             scp -i /myssh/id_rsa -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null ${ts}.sh root@$HOST:/tmp/${ts}.sh
             ssh -i /myssh/id_rsa -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null root@$HOST "/tmp/${ts}.sh"          
```

```suggestion
            ssh -i /myssh/id_rsa -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null root@$HOST "/tmp/${ts}.sh; rm /tmp/${ts}.sh"          
```
Could you do this so that 2 ssh calls are not needed? Same for other commands.

---

Sure , Thanks for Suggestion